cobalt/devinfra/kokoro/bin/dind_build.sh

```diff
@@ -42,17 +42,28 @@ fi
 # Load common routines (we only need run_gn and ninja_build).
 . $(dirname "$0")/common.sh
 
-# Using repository root as work directory.
 WORKSPACE_COBALT="${KOKORO_ARTIFACTS_DIR}/git/src"
-cd "${WORKSPACE_COBALT}"
 
-configure_environment
-
-# Build Cobalt.
 pipeline () {
   local out_dir="${WORKSPACE_COBALT}/out/${TARGET_PLATFORM}_${CONFIG}"
-  run_gn "${out_dir}" "${TARGET_PLATFORM}" "${EXTRA_GN_ARGUMENTS:-}"
-  ninja_build "${out_dir}" "${TARGET}"
+  local src_dir="${KOKORO_ARTIFACTS_DIR}/chromium/src"
+  mkdir -p "${src_dir}"
+  cp -aT "${WORKSPACE_COBALT}" "${src_dir}"  # avoid CIPD invalid cross-device link
+
+  cd "${KOKORO_ARTIFACTS_DIR}/chromium"
+  git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git tools/depot_tools
```

Comment:
there might not be a need for this specific line, since Kokoro can run checkout steps

go/kokoro-multi-scm-config#example

Reply:
this is similar to how github actions works, so this is preferred